src/server/server.js

```diff
@@ -1,87 +1,39 @@
 const express = require('express');
 const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser');
+const session = require('express-session');
 const morgan = require('morgan');
+const mongoose = require('mongoose');
 const app = express();
-const router = express.Router();
+const router = require('./router');
+const passport = require('passport');
 
-const path = require('path');
-const fs = require('fs');
-const util = require('util');
+const config = require('./config');
 
-const readFileAsync = util.promisify(fs.readFile);
-const writeFileAsync = util.promisify(fs.writeFile);
+mongoose.Promise = Promise;
 
-const storageFilePath = path.resolve(__dirname, 'data.json');
+mongoose.connect(config.mongodb.connectionUrl, { useNewUrlParser: true });
```

Comment:
connect method returns promise, please make some notifications in then and catch methods